src/main/perl/Filesystem.pm

```diff
@@ -157,6 +157,67 @@ sub remove_if_needed
 
 =pod
 
+=head2 check_in_fstab
+
+Parse the fstab entry for the C<$self> filesystem. When a (PART)UUID is found, 
+this is used instead of the device name or label. 
+If it concerns a protected (i.e. never-modify) mountpoint or filesystem type,
+it will add it, but changes will not be allowed.
+When the entry should be added or inserted, this sub will return the device name to use,
+otherwise 0 is returned.
```

Comment:
better to return `undef (or simply `return;`) in case of failure, so you can unambigoiiusly test for failure with `if defined`